core/src/main/java/org/lflang/federated/launcher/FedLauncherGenerator.java

```diff
@@ -382,7 +364,11 @@ private String getLaunchCode(String rtiLaunchCode) {
         "# The RTI will be brought back to foreground",
         "# to be responsive to user inputs after all federates",
         "# are launched.",
-        rtiLaunchCode,
+        "if [ \"$1\" = \"-l\" ]; then",
```

Comment:
If you use `"""` you don't have to escape and it's easier to read.

Reply:
Not going to do this now. This should be done throughout this file.